### PR TITLE
Add options overload parameter to sendFile function (#238)

### LINKED ISSUE #238: Add options overload parameter to sendFile function
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

Sometimes it's required to control cache headers more granularly, so it would be nice to have an option to overwrite the options while calling `sendFile` (just like it's already done for `download`).

### Motivation

_No response_

### Example

```js
fastify.get('/path', function (req, reply) {
  return reply.sendFile('some-file.js', { maxAge: 0 })
})

fastify.get('/path/custom', function (req, reply) {
  return reply.sendFile('some-file.js', 'custom-filename.js', { maxAge: 0 })
})
```

### PR DESCRIPTION
**Pull request by mav-rik**
Fixes #238

#### Checklist

- [X] run `npm run test` and `npm run benchmark`
- [X] tests and/or benchmarks are included
- [X] documentation is changed or added
- [X] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — CHANGES_REQUESTED** [review]
Thanks! Could you update the README as well?

**Comment by mav-rik** [comment]
@mcollina done

**Review by mcollina — APPROVED** [review]
lgtm

File: index.js [hunk 1]
```diff
@@ -235,12 +235,16 @@ async function fastifyStatic (fastify, opts) {
   }
 
   if (opts.decorateReply !== false) {
-    fastify.decorateReply('sendFile', function (filePath, rootPath) {
+    fastify.decorateReply('sendFile', function (filePath, rootPath, options = {}) {
```
Comment (Eomm) [inline index.js:238]: The default value is not necessary

```suggestion
    fastify.decorateReply('sendFile', function (filePath, rootPath, options) {
```

**Review by Eomm — COMMENTED** [review]
Just a hint

**Review by mcollina — APPROVED** [review]
lgtym

**Comment by mav-rik** [comment]
@mcollina is it ready to be merged? Any more approves needed?

**Comment by mcollina** [comment]
I just needed to ship a security release for this module. Landing this now.

**Pull request merged**